### PR TITLE
Added spinner while submitting the form

### PR DESCRIPTION
**Pull request by nitinrgupta**
##### Description

Added spinner to submit button once the user clicks on it unitl the API call is finished

##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)

UI 

##### Refers/Fixes

Fixes: #259

### CODE REVIEW

File: src/components/MarketSubscriberForm.tsx [hunk 1]
```diff
@@ -3,9 +3,19 @@ import { Button, Form, Input, message, Modal } from 'antd';
 import styled from 'styled-components';
 import fetch from 'isomorphic-fetch';
 import env from '../constants/environment';
+import { FormComponentProps } from 'antd/lib/form/Form';
 
 const FormItem = Form.Item;
 
+interface Props {
```
Comment (eswarasai) [inline src/components/MarketSubscriberForm.tsx]: @nitinrgupta -- Rename this to `SubscriptionPopUpProps` which extends `FormComponentProps` and use it directly below

Comment (nitinrgupta) [reply src/components/MarketSubscriberForm.tsx]: Done @eswarasai

**Comment by nitinrgupta** [comment]
@eswarasai we good to merge this?

**Comment by nitinrgupta** [comment]
@eswarasai I don't see the build failing. Did you get any error?

**Comment by eswarasai** [comment]
@nitinrgupta -- Below is the error message

```
ReferenceError: Failed exporting HTML for URL / (src/containers/Home): window is not defined
```

Try removing the `node_modules` and doing a clean npm install.

**Comment by nitinrgupta** [comment]
@eswarasai tried removing `node_modules` and doing a clean npm install. Still didn't face any issues while creating a build locally. This was on top of my `/partners` page changes. 

Can you check once again whether the build issues is still existing?

**Comment by eswarasai** [comment]
@nitinrgupta -- `npm run build` is still failing for me 🤷‍♂️

**Comment by eswarasai** [comment]
@nitinrgupta -- The build is still failing for me because of this issue which I'm aware of but weird that you're able to do the build, so does on Travis CI. Anyway, I've just approved the PR in order to move things forward. Feel free to merge it whenever you'd like to.
Ref: https://react-static.js.org/docs/concepts/#writing-universal-node-safe-code

**Pull request merged**